tools/sloths/src/utils/ru-noun-ending.ts

```diff
@@ -0,0 +1,16 @@
+export const ruNounEnding = (num: number, string1: string, string2: string, stringN: string): string => {
```

Comment:
Function names should be verbs

Reply:
In this case, need to use the default pluralization rule in Vue I18n. I will remove this utility